app/src/sharedTest/java/org/oppia/android/app/options/OptionsFragmentTest.kt

```diff
@@ -207,31 +208,56 @@ class OptionsFragmentTest {
   }
 
   @Test
-  fun testOptionsFragment_readingTextSize_testOnActivityResult() {
+  fun testOptionsFragment_defaultReadingTextSizeIsDisplayed() {
     launch<OptionsActivity>(
       createOptionActivityIntent(
         internalProfileId = 0,
         isFromNavigationDrawer = true
       )
     ).use {
-      val resultDataIntent = Intent()
-      resultDataIntent.putExtra(MESSAGE_READING_TEXT_SIZE_ARGUMENT_KEY, "Large")
-      val activityResult = ActivityResult(Activity.RESULT_OK, resultDataIntent)
-
-      val activityMonitor = getInstrumentation().addMonitor(
-        ReadingTextSizeActivity::class.java.name,
-        activityResult,
-        true
+      testCoroutineDispatchers.runCurrent()
+      onView(
+        atPositionOnView(
+          recyclerViewId = R.id.options_recyclerview,
+          position = 0,
+          targetViewId = R.id.reading_text_size_text_view
+        )
+      ).check(
+        matches(withText("Medium"))
       )
+    }
+  }
 
-      it.onActivity { activity ->
-        activity.startActivityForResult(
-          createReadingTextSizeActivityIntent(summaryValue = "Small"),
-          REQUEST_CODE_TEXT_SIZE
+  @Test
+  fun testOptionsFragment_configChange_defaultReadingTextSizeIsDisplayed() {
+    launch<OptionsActivity>(
+      createOptionActivityIntent(
+        internalProfileId = 0,
+        isFromNavigationDrawer = true
+      )
+    ).use {
+      rotateToLandscape()
+      onView(
+        atPositionOnView(
+          recyclerViewId = R.id.options_recyclerview,
+          position = 0,
+          targetViewId = R.id.reading_text_size_text_view
         )
-      }
+      ).check(
+        matches(withText("Medium"))
+      )
+    }
+  }
 
-      getInstrumentation().waitForMonitorWithTimeout(activityMonitor, 3)
+  @Test
+  @Config(qualifiers = "sw600dp")
```

Comment:
please add espresso test result for these test too, on tablet.

Reply:
Done.